src/test/regress/sql/upgrade_pg_dist_object_test_before.sql

```diff
@@ -0,0 +1,30 @@
+-- create some objects that we just included into distributed object
+-- infrastructure in 9.1 versions but not included in 9.0.2
+
+-- extension propagation --
+
+-- create an extension on all nodes and a table that depends on it
+CREATE EXTENSION isn;
+SELECT run_command_on_workers($$CREATE EXTENSION isn;$$);
+
+CREATE TABLE isn_dist_table (key int, value issn);
+SELECT create_reference_table('isn_dist_table');
+
+-- create an extension on all nodes, but do not create a table depending on it
+CREATE EXTENSION seg;
+SELECT run_command_on_workers($$CREATE EXTENSION seg;$$);
+
+-- schema propagation --
+
+-- public schema
+CREATE TABLE dist_table (a int);
+SELECT create_reference_table('dist_table');
+
+-- custom schema
+CREATE SCHEMA new_schema;
+SELECT run_command_on_workers($$CREATE SCHEMA new_schema;$$);
```

Comment:
Not sure this is useful since we already captured non-public schemas as of 9.0, so the upgrade logic will not affect that.

Reply:
makes sense, actually I was not in need of such a command, removing